layouts/events/list.html

```diff
@@ -18,60 +18,63 @@ <h1>Upcoming Events</h1>
                 <h2 id="event-list-heading" class="mb-8">All Upcoming Events</h2>
                 <ul id="event-list" class="list-none md:flex md:flex-wrap pl-0">
                     {{ $events := (where $.Pages "Type" "events") }}
-                    {{ range $index, $event := (sort $events ".Params.event.start_date") }}
-                        <li class="w-full m-0 p-0" data-event-type='{{ delimit .Params.event.type "," }}'>
-                            <article class="{{ if lt (add $index 1) (len $events) }} pb-16 mb-12 border-b border-gray-300 {{ end }}">
-                                <h3>
-                                    {{ if .Params.url_slug }}
-                                        <a class="text-2xl" href="/events/{{ .Params.url_slug }}/">
-                                            {{ .Params.title }}
-                                        </a>
-                                    {{ else }}
-                                        <a class="text-2xl flex items-center" href="{{ .Params.event.registration_url }}" target="_blank">
-                                            {{ .Params.title }}
-                                            <i class="text-sm ml-2 fas fa-external-link-alt"></i>
-                                        </a>
-                                    {{ end }}
-                                </h3>
-                                <div>
-                                    <i class="text-gray-800 far fa-calendar inline-block"></i>
-                                    <span class="ml-1 text-sm uppercase">
-                                        {{ if eq .Params.event.start_date .Params.event.end_date }}
-                                            {{ dateFormat "Monday, Jan 2" .Params.event.start_date }}
+                    {{ $currentEvents := (where $events "{{ (time .Params.event.start_date).After (now.AddDate 0 0 -1) }}" "ne" true ) }}
```

Comment:
Since we aren't passing this as a list to a paginator, we can detangle this a bit by just doing the filtering as a separate if:

```
{{ range $index, $event ... }}
    {{ if (time .Params.event.start_date).After (now.AddDate 0 0 -1) }}
```

Reply:
`$currentEvents` was leftover from when I was trying to filter using `where` so I would need the `if..` statement. I went ahead and removed the old code to simplify it all. Thanks for catching this!